code/game/objects/items/crayons.dm

```diff
@@ -104,7 +128,7 @@
 	return
 
 /obj/item/pen/crayon/attack(mob/user, var/target_zone)
-	if(ishuman(user))
+	if(ishuman(user) && utensiltype == "crayon")
```

Comment:
I don't see how you deciding to draw an arrow would prevent you from eating a crayon

Reply:
It doesn't.  utensiltype is used to pass on whether the obj/item/pen/crayon is a crayon, or a piece of chalk (a subtype of crayon).  Can't eat chalk, only crayons.